lib/src/presentation/widget/favorites_list.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/material.dart';
+import 'package:hearthstoneapp/src/core/util/api_response_state.dart';
+import 'package:hearthstoneapp/src/presentation/bloc/favorites_bloc.dart';
+import 'package:hearthstoneapp/src/presentation/widget/card_list_container.dart';
+import 'package:hearthstoneapp/src/presentation/widget/not_found.dart';
+
+class FavoritesList extends StatelessWidget {
+  final FavoritesBloc favoritesBloc;
+
+  const FavoritesList(this.favoritesBloc);
+
+  @override
+  Widget build(BuildContext context) {
+    return StreamBuilder<ApiResponse>(
+      key: UniqueKey(),
+      stream: favoritesBloc.getCardListStream(),
+      builder: (context, snapshot) {
```

Comment:
Trailling comma and specify context/snapshot types

Reply:
Done